extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueLightHandler.java

```diff
@@ -405,7 +405,7 @@ public void onLightStateChanged(@Nullable HueBridge bridge, FullLight fullLight)
         updateState(CHANNEL_COLOR, hsbType);
 
         ColorMode colorMode = fullLight.getState().getColorMode();
-        if (colorMode != null && colorMode.equals(ColorMode.CT)) {
+        if (colorMode != null && ColorMode.CT.equals(colorMode)) {
```

Comment:
is the null check still required when you turn around the `equals` call?

Reply:
No, it is not.